app/models/registration-schema.ts

```diff
@@ -57,6 +57,7 @@ export default class RegistrationSchemaModel extends OsfModel {
     @attr('boolean') active!: boolean;
     @attr('fixstring') name!: string;
     @attr('number') schemaVersion!: number;
+    @attr('string') groupType!: string;
```

Comment:
Is this right? I don think there is a `groupType` attribute on a `RegistrationSchema` model.

Reply:
Yeah, doesn't make sense to me either.